server/channels/app/scheduled_post_test.go

```diff
@@ -578,3 +578,86 @@ func TestUpdateScheduledPost(t *testing.T) {
 		require.Equal(t, model.ScheduledPostErrorUnknownError, createdScheduledPost.ErrorCode)
 	})
 }
+
+func TestDeleteScheduledPost(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	t.Run("base case", func(t *testing.T) {
+		// first we'll create a scheduled post
+		scheduledPost := &model.ScheduledPost{
+			Draft: model.Draft{
+				CreateAt:  model.GetMillis(),
+				UserId:    th.BasicUser.Id,
+				ChannelId: th.BasicChannel.Id,
+				Message:   "this is a scheduled post",
+			},
+			ScheduledAt: model.GetMillis() + 100000, // 100 seconds in the future
+		}
+		createdScheduledPost, appErr := th.App.SaveScheduledPost(th.Context, scheduledPost)
+		require.Nil(t, appErr)
+		require.NotNil(t, createdScheduledPost)
+
+		fetchedScheduledPost, err := th.Server.Store().ScheduledPost().Get(scheduledPost.Id)
+		require.NoError(t, err) // This will produce error as the row doesn't exist
```

Comment:
Same comment issue as the other comment.

Reply:
Fixed.